server/routes/victorops/tasks.py

```
@@ -0,0 +1,432 @@
+                                "[VICTOROPS] Triggered RCA for incident %s (task=%s)",
+                                incident_db_id, task.id,
+                            )
+                    except Exception as e:
+                        logger.warning("[VICTOROPS] Failed to trigger RCA: %s", e)
+
+                logger.info(
+                    "[VICTOROPS] Processed %s event for incident #%s (db_id=%s)",
+                    alert_phase, incident_number, incident_db_id,
+                )
+
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)